test/dfusionMultiSafes.js

```diff
@@ -23,6 +22,43 @@ const {
 } = require("../scripts/utils/trading_strategy_helpers")
 const { waitForNSeconds, toETH, execTransaction, deploySafe } = require("../scripts/utils/internals")
 
+const checkPricesOfBracketStrategy = async function(targetPrice, slaveSafes, exchange) {
+  const rangePercentage = 0.2
+  const stepSize = (targetPrice * (2 * rangePercentage)) / slaveSafes.length
+  const minimalPrice = targetPrice * (1 - rangePercentage)
+  let multiplicator = new BN("10")
+  if (targetPrice < 10) {
+    multiplicator = new BN("10000000")
+  }
+  // Correctness assertions
+  for (const [index, slaveAddress] of slaveSafes.entries()) {
+    const auctionElements = exchangeUtils.decodeOrdersBN(await exchange.getEncodedUserOrders(slaveAddress))
+    assert.equal(auctionElements.length, 2)
+    const [buyOrder, sellOrder] = auctionElements
+    // Check buy order prices
```

Comment:
Sorry to bring up questions on old changes, but these comments are too vague to know what is being"checked here". Perhaps the all the comments before assertions could be expanded upon and included as failure strings as the optional third parameter of the assertions themselves.

As a new developer coming into this project, this comment contains very little context apart from the fact that the next line is an assertion involving buy order price.

Reply:
>an assertion involving buy order price.
It is not just involving buy order price, it is checking the actual buy order price. It checks the order price against the recalculated intended order price. 

How would you have described it in the comments?

Reply:
That is the problem here. I don't know what this is for, so I can't describe it and the message doesn't help....

Reply:
What about the buy order price is being checked? is it non-zero?